<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DlgTextEdit</class>
 <widget class="QDialog" name="DlgTextEdit">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>446</width>
    <height>291</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>Sans Serif</family>
    <pointsize>9</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>Edit text</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>2</number>
   </property>
   <property name="margin">
    <number>4</number>
   </property>
   <item>
    <layout class="QGridLayout" name="gridLayout">
     <property name="margin">
      <number>0</number>
     </property>
     <property name="spacing">
      <number>2</number>
     </property>
     <item row="0" column="0" colspan="2">
      <widget class="QLabel" name="label">
       <property name="font">
        <font>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Properties for all shots</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_2">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>10</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>10</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="text">
        <string notr="true"/>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <property name="spacing">
        <number>2</number>
       </property>
       <property name="margin">
        <number>0</number>
       </property>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_12">
         <property name="spacing">
          <number>0</number>
         </property>
         <property name="margin">
          <number>0</number>
         </property>
         <item>
          <widget class="QFontComboBox" name="fontStyleCB">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>24</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>24</height>
            </size>
           </property>
           <property name="toolTip">
            <string>Select font family</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="fontSize">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="minimumSize">
            <size>
             <width>48</width>
             <height>24</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>48</width>
             <height>24</height>
            </size>
           </property>
           <property name="toolTip">
            <string>Select font size</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="cCustomColorComboBox" name="FontColorCombo">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="minimumSize">
            <size>
             <width>48</width>
             <height>24</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>48</width>
             <height>24</height>
            </size>
           </property>
           <property name="toolTip">
            <string>Select font color</string>
           </property>
           <property name="editable">
            <bool>false</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_7">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="minimumSize">
            <size>
             <width>10</width>
             <height>24</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>10</width>
             <height>24</height>
            </size>
           </property>
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="fontEffectCB">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="minimumSize">
            <size>
             <width>44</width>
             <height>24</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>44</width>
             <height>24</height>
            </size>
           </property>
           <property name="toolTip">
            <string>Select font effect (outerline, shadow)</string>
           </property>
           <property name="styleSheet">
            <string notr="true">QComboBox QAbstractItemView {
background: #ffffff;
color: #000000;
}</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="cCustomColorComboBox" name="StyleShadowColorCombo">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="minimumSize">
            <size>
             <width>48</width>
             <height>24</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>48</width>
             <height>24</height>
            </size>
           </property>
           <property name="toolTip">
            <string>Select outerline/shadow color</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <property name="spacing">
          <number>0</number>
         </property>
         <property name="margin">
          <number>0</number>
         </property>
         <item>
          <widget class="QToolButton" name="bold">
           <property name="minimumSize">
            <size>
             <width>22</width>
             <height>24</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>22</width>
             <height>24</height>
            </size>
           </property>
           <property name="toolTip">
            <string>Set text to Bold</string>
           </property>
           <property name="text">
            <string notr="true">...</string>
           </property>
           <property name="icon">
            <iconset>
             <normaloff>img/text_bold.png</normaloff>img/text_bold.png</iconset>
           </property>
           <property name="checkable">
            <bool>true</bool>
           </property>
           <property name="checked">
            <bool>false</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QToolButton" name="Italic">
           <property name="minimumSize">
            <size>
             <width>22</width>
             <height>24</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>22</width>
             <height>24</height>
            </size>
           </property>
           <property name="toolTip">
            <string>Set text to Italic</string>
           </property>
           <property name="text">
            <string notr="true">...</string>
           </property>
           <property name="icon">
            <iconset>
             <normaloff>img/text_italic.png</normaloff>img/text_italic.png</iconset>
           </property>
           <property name="checkable">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QToolButton" name="Souligne">
           <property name="minimumSize">
            <size>
             <width>22</width>
             <height>24</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>22</width>
             <height>24</height>
            </size>
           </property>
           <property name="toolTip">
            <string>Set text to Underline</string>
           </property>
           <property name="text">
            <string notr="true">...</string>
           </property>
           <property name="icon">
            <iconset>
             <normaloff>img/text_underline.png</normaloff>img/text_underline.png</iconset>
           </property>
           <property name="checkable">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_6">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="minimumSize">
            <size>
             <width>10</width>
             <height>24</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>10</width>
             <height>24</height>
            </size>
           </property>
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QToolButton" name="textLeft">
           <property name="minimumSize">
            <size>
             <width>22</width>
             <height>24</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>22</width>
             <height>24</height>
            </size>
           </property>
           <property name="toolTip">
            <string>Align left</string>
           </property>
           <property name="text">
            <string notr="true">...</string>
           </property>
           <property name="icon">
            <iconset>
             <normaloff>img/text_align_left.png</normaloff>img/text_align_left.png</iconset>
           </property>
           <property name="checkable">
            <bool>true</bool>
           </property>
           <property name="checked">
            <bool>true</bool>
           </property>
           <property name="autoExclusive">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QToolButton" name="textCenter">
           <property name="minimumSize">
            <size>
             <width>22</width>
             <height>24</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>22</width>
             <height>24</height>
            </size>
           </property>
           <property name="toolTip">
            <string>Align center</string>
           </property>
           <property name="text">
            <string notr="true">...</string>
           </property>
           <property name="icon">
            <iconset>
             <normaloff>img/text_align_center.png</normaloff>img/text_align_center.png</iconset>
           </property>
           <property name="checkable">
            <bool>true</bool>
           </property>
           <property name="autoExclusive">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QToolButton" name="textJustif">
           <property name="minimumSize">
            <size>
             <width>22</width>
             <height>24</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>22</width>
             <height>24</height>
            </size>
           </property>
           <property name="toolTip">
            <string>Align justify</string>
           </property>
           <property name="text">
            <string notr="true">...</string>
           </property>
           <property name="icon">
            <iconset>
             <normaloff>img/text_align_justif.png</normaloff>img/text_align_justif.png</iconset>
           </property>
           <property name="checkable">
            <bool>true</bool>
           </property>
           <property name="autoExclusive">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QToolButton" name="textRight">
           <property name="minimumSize">
            <size>
             <width>22</width>
             <height>24</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>22</width>
             <height>24</height>
            </size>
           </property>
           <property name="toolTip">
            <string>Align right</string>
           </property>
           <property name="text">
            <string notr="true">...</string>
           </property>
           <property name="icon">
            <iconset>
             <normaloff>img/text_align_right.png</normaloff>img/text_align_right.png</iconset>
           </property>
           <property name="checkable">
            <bool>true</bool>
           </property>
           <property name="autoExclusive">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_12">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="minimumSize">
            <size>
             <width>10</width>
             <height>24</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>10</width>
             <height>24</height>
            </size>
           </property>
           <property name="text">
            <string notr="true"/>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QToolButton" name="textUp">
           <property name="minimumSize">
            <size>
             <width>22</width>
             <height>24</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>22</width>
             <height>24</height>
            </size>
           </property>
           <property name="toolTip">
            <string>Align top</string>
           </property>
           <property name="text">
            <string notr="true">...</string>
           </property>
           <property name="icon">
            <iconset>
             <normaloff>img/text_align_vup.png</normaloff>img/text_align_vup.png</iconset>
           </property>
           <property name="checkable">
            <bool>true</bool>
           </property>
           <property name="autoExclusive">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QToolButton" name="textVCenter">
           <property name="minimumSize">
            <size>
             <width>22</width>
             <height>24</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>22</width>
             <height>24</height>
            </size>
           </property>
           <property name="toolTip">
            <string>Align in the middle</string>
           </property>
           <property name="text">
            <string notr="true">...</string>
           </property>
           <property name="icon">
            <iconset>
             <normaloff>img/text_align_vcenter.png</normaloff>img/text_align_vcenter.png</iconset>
           </property>
           <property name="checkable">
            <bool>true</bool>
           </property>
           <property name="autoExclusive">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QToolButton" name="textBottom">
           <property name="minimumSize">
            <size>
             <width>22</width>
             <height>24</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>22</width>
             <height>24</height>
            </size>
           </property>
           <property name="toolTip">
            <string>Align bottom</string>
           </property>
           <property name="text">
            <string notr="true">...</string>
           </property>
           <property name="icon">
            <iconset>
             <normaloff>img/text_align_vbottom.png</normaloff>img/text_align_vbottom.png</iconset>
           </property>
           <property name="checkable">
            <bool>true</bool>
           </property>
           <property name="checked">
            <bool>false</bool>
           </property>
           <property name="autoExclusive">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_16">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>24</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>24</height>
            </size>
           </property>
           <property name="text">
            <string notr="true"/>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QPlainTextEdit" name="plainTextEdit">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>80</height>
          </size>
         </property>
         <property name="verticalScrollBarPolicy">
          <enum>Qt::ScrollBarAlwaysOn</enum>
         </property>
         <property name="horizontalScrollBarPolicy">
          <enum>Qt::ScrollBarAlwaysOn</enum>
         </property>
         <property name="documentTitle">
          <string notr="true"/>
         </property>
         <property name="plainText">
          <string notr="true"/>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <widget class="Line" name="line">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_5">
     <property name="spacing">
      <number>0</number>
     </property>
     <property name="margin">
      <number>0</number>
     </property>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="OKBT">
       <property name="toolTip">
        <string>Accept modification and exit dialog box</string>
       </property>
       <property name="text">
        <string>&amp;Ok</string>
       </property>
       <property name="icon">
        <iconset>
         <normaloff>img/action_ok.png</normaloff>img/action_ok.png</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="CloseBT">
       <property name="toolTip">
        <string>Cancel modification and exit dialog box</string>
       </property>
       <property name="text">
        <string>&amp;Cancel</string>
       </property>
       <property name="icon">
        <iconset>
         <normaloff>img/action_cancel.png</normaloff>img/action_cancel.png</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="HelpBT">
       <property name="toolTip">
        <string>Open help page describing this dialog box</string>
       </property>
       <property name="text">
        <string>&amp;Help</string>
       </property>
       <property name="icon">
        <iconset>
         <normaloff>img/help.png</normaloff>img/help.png</iconset>
       </property>
       <property name="shortcut">
        <string>F1</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>cCustomColorComboBox</class>
   <extends>QComboBox</extends>
   <header>cCustomComboBox.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>fontStyleCB</tabstop>
  <tabstop>fontSize</tabstop>
  <tabstop>FontColorCombo</tabstop>
  <tabstop>fontEffectCB</tabstop>
  <tabstop>StyleShadowColorCombo</tabstop>
  <tabstop>bold</tabstop>
  <tabstop>Italic</tabstop>
  <tabstop>Souligne</tabstop>
  <tabstop>textLeft</tabstop>
  <tabstop>textCenter</tabstop>
  <tabstop>textJustif</tabstop>
  <tabstop>textRight</tabstop>
  <tabstop>textUp</tabstop>
  <tabstop>textVCenter</tabstop>
  <tabstop>textBottom</tabstop>
  <tabstop>OKBT</tabstop>
  <tabstop>CloseBT</tabstop>
  <tabstop>HelpBT</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
